src/renderer/services/file-management-system/test/file-management-system.test.ts

```diff
@@ -576,4 +576,14 @@ describe("FileManagementSystem", () => {
       await expect(fms.cancel(mockUploadId)).rejectedWith(Error);
     });
   });
+
+  describe("Dynamically set chunksInFlight according to chunkSize", () => {
```

Comment:
This is kinda a nit but it breaks the pattern the other tests in this repo have where describe is used to establish the method being tested and then the inner "it"s are the tests. Like so:
```
describe("chunksInFlightMaxForChunkSize", () => {
    [
         { "chunkSize": 50000000, "expectedChunksInFlight": 20 },
         ....(the other test cases)
     ].forEach(({ chunkSize, expectedChunksInFlight }) => {
          it(`Dynamically sets chunksInFlight to ${expectedOutput} for chunkSize ${input}`, () => {
                  expect(FileManagementSystem.chunksInFlightMaxForChunkSize(chunkSize)).to.equal(expectedChunksInFlight);
           })
     })
})
```